RestSharp.Tests/JsonTests.cs

```diff
@@ -1060,7 +1062,7 @@ public string CreateDynamicJsonStringDictionary()
 
         private static T GetPayLoad<T>(string fileName)
         {
-            string doc = File.ReadAllText(Path.Combine("SampleData", fileName));
+            string doc = File.ReadAllText(Path.Combine(currentPath, "SampleData", fileName));
```

Comment:
Why is this change neccesary?

Reply:
NUnit3 console runner does not guarantee to execute in the test library location. As a result, none of the test files are found by tests. You can try yourself or resolve it differently, I find this way the most efficient.

Reply:
👍